tests/fixtures/ember-container-query-javascript/input/package.json

```diff
@@ -105,10 +105,9 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-qunit": "^7.3.4",
     "eslint-plugin-simple-import-sort": "^8.0.0",
-    "eslint-plugin-typescript-sort-keys": "^2.1.0",
     "lerna-changelog": "^2.2.0",
     "loader.js": "^4.7.0",
-    "prettier": "^2.8.2",
```

Comment:
I made a mistake of not keeping the installed versions the same between the JavaScript and TypeScript versions of `ember-container-query` (the copied addon).

If I have time, I'll later review and standardize the installed versions in a separate pull request.

Reply:
Addressed in #8.